sumologic/sumologic_client.go

```diff
@@ -186,9 +186,6 @@ func (s *Client) PostRawPayload(urlPath string, payload string) ([]byte, error)
 }
 
 func (s *Client) Put(urlPath string, payload interface{}, isAdminMode bool) ([]byte, error) {
-	SumoMutexKV.Lock(urlPath)
```

Comment:
This was serializing update calls to a resource. Now we may process 2 update requests at the same time. This may be okay provided we process the requests appropriately in the backend. We should keep an eye out for issue around update calls.